week-1-ts-ecosystem/w01e03-object-oriented-structure/src/Products.ts

```diff
@@ -0,0 +1,81 @@
+export enum ProductType {
+  "BuyNow" = "Buy Now",
+  "Auction" = "Auction",
+  "Free" = "Free",
+}
```

Comment:
skoro zapis `value / key` jest `1:1` to w mojej opinii wystarczy tutaj:

`type ProductType = "Boy Now" | "Auction" | "Free"`